bin/tape

```diff
@@ -1,12 +1,34 @@
 #!/usr/bin/env node
 
-var path = require('path');
+var resolveModule = require('resolve').sync;
+var resolvePath = require('path').resolve;
+var parseOpts = require('minimist');
 var glob = require('glob');
 
-process.argv.slice(2).forEach(function (arg) {
+var opts = parseOpts(process.argv.slice(2), {
+        alias: { 'r': 'require' },
+        string: 'require',
+        default: {'r': [] }
+    });
+
+var cwd = process.cwd();
+
+if (typeof opts.require === 'string') {
```

Comment:
you could replace this if block with `var requires = [].concat(opts.require)`, fwiw (that would also avoid mutating `opts`)